app/src/main/java/org/greenstand/android/TreeTracker/di/ViewModelModule.kt

```diff
@@ -0,0 +1,16 @@
+package org.greenstand.android.TreeTracker.di
+
+import org.greenstand.android.TreeTracker.dashboard.DashboardViewModel
+import org.greenstand.android.TreeTracker.languagepicker.LanguagePickerViewModel
+import org.greenstand.android.TreeTracker.signup.SignupViewModel
+import org.greenstand.android.TreeTracker.splash.SplashScreenViewModel
+import org.koin.android.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val viewModelModule = module {
```

Comment:
This is already done in the app module, but this is cleaner. For now, we can delete this and break up the app module is a different PR

Reply:
Ah okay, that works! Maybe I'll do that this weekend for funsies. For this PR I'll move things over and nuke it